README.md

```diff
@@ -562,35 +562,80 @@ The bottom line is that the only way to confirm that a transaction is successful
 is to check the hash against the chain.
 Also, the block containing the transaction has to be finalized for the transaction to be as well.
 
-## Tools
+## Testing
 
-### Transfer client
+### Running Rosetta CLI for testing
```

Comment:
Consider mentioning why we forked the CLI. Which is to make the tool understand account aliases, i.e. that a transaction affecting the balance of an account affects all aliases of that account as well.

The test will fail if run with the official CLI.